pkg/trie/triedb/node.go

```diff
@@ -110,6 +110,7 @@ func inMemoryFetchedValue(value nodeValue, prefix []byte, db db.DBGetter) ([]byt
 
 type Node interface {
 	isNode()
+	getPartialKey() []byte
```

Comment:
do we still need this if you now have a method to implement?
```suggestion
	getPartialKey() []byte
```